go/vt/vtgate/buffer/flags_test.go

```diff
@@ -20,45 +20,60 @@ import (
 	"flag"
 	"strings"
 	"testing"
+
+	"github.com/spf13/pflag"
 )
 
 func TestVerifyFlags(t *testing.T) {
+	parse := func(args []string) {
+		fs := pflag.NewFlagSet("vtgate_buffer_test", pflag.ContinueOnError)
+		registerFlags(fs)
+
+		if err := fs.Parse(args); err != nil {
+			t.Errorf("failed to parse args %v: %s", args, err)
+		}
+	}
 	resetFlagsForTesting := func() {
 		// Set all flags to their default value.
```

Comment:
is this comment still valid?

Reply:
yes, parsing the flags will set them to back to the default value.